### ISSUE TITLE
[Feature request] mempool info - recommended fees

### ISSUE THREAD

**Issue by openoms** [dennisreimann/kindle-display#17]
using the mempool.space (or self-hosted) API the Kindle could display the recommended fees like:
https://mempool.space/api/v1/fees/recommended
![image](https://user-images.githubusercontent.com/43343391/124455976-1869b580-dd82-11eb-9b98-27e44761d64a.png)

This is often more interesting to me than the price so could take over at least the place of one currency maybe.

**Comment by dennisreimann** [comment]
Good idea. In general I'd like to offer more display options, so that one could also cycle through them randomly.
